src/plugins/navcurrent/navcurrent.js

```diff
@@ -74,14 +74,15 @@ var componentName = "wb-navcurr",
 						// Pre-process the breadcrumb links
 						localBreadcrumbLinksArray = [];
 						localBreadcrumbLinksUrlArray = [];
-						localBreadcrumbLinks = ( breadcrumb.jquery ? breadcrumb[ 0 ] : breadcrumb ).getElementsByTagName( "a" );
+						localBreadcrumbLinks = ( breadcrumb.jquery ? breadcrumb[ 0 ] : breadcrumb ).getElementsByTagName( "li" );
 						len = localBreadcrumbLinks.length;
-						for ( i = 0; i !== len; i += 1 ) {
-							link = localBreadcrumbLinks[ i ];
-							linkHref = link.getAttribute( "href" );
-							if ( linkHref.length !== 0 && linkHref.charAt( 0 ) !== "#" ) {
-								localBreadcrumbLinksArray.push( link );
-								localBreadcrumbLinksUrlArray.push( link.hostname + link.pathname.replace( /^([^\/])/, "/$1" ) );
+						if ( len && len > 0 ) {
+							link = localBreadcrumbLinks[ ( len - 1 ) ];
+							child = link.firstChild;
+							linkHref = ( child ) ? child.getAttribute( "href" ) : "";
```

Comment:
You may want to test is the first element is really an anchor. Before that action was implicitly done at line 77
```
linkHref = ( child && child.nodeName === "A" ) ? child.getAttribute( "href" ) : "";
```